go.mod

```diff
@@ -45,6 +45,7 @@ require (
 	go.uber.org/atomic v1.7.0
 	go.uber.org/zap v1.17.0
 	golang.org/x/exp v0.0.0-20200224162631-6cc2880d07d6
+	golang.org/x/sys v0.0.0-20210902050250-f475640dd07b
```

Comment:
can you use pstuil ? psutil has been used in milvus.

Reply:
OK，I agree